playbooks/roles/airship-deploy-tempest/files/tempest_blacklist

```diff
@@ -1,3 +1,7 @@
 #  Add tests to be blacklisted below using the following format:
 #  - (?:tempest\.api\.identity\.admin\.v3\.test_groups\.GroupsV3TestJSON\.test_list_groups)
 - (?:tempest\.api\.identity\.v3\.test_domains\.DefaultDomainTestJSON\.test_default_domain_exists)
+- (?:tempest\.api\.compute\.servers\.test_server_actions\.ServerActionsTestJSON\.test_reboot_server_hard)
```

Comment:
are you going to add the failed test cases from full test suite as well?

Reply:
Sure, but since I don't trust my own dev environment to produce an accurate list (I'm still having problems getting nova instances to run), I'd like to get the full list of failures from QE since they're more likely to be "real" failures.